### PR TITLE
Misc psion armor fixes/tweaks

### PR DESCRIPTION
**Pull request by benj8560**
Fixes psion heavy armor preventing its own gloves from spawning
Fixes the psion light armor helmet having its helmet light icon in the incorrect place and being incorrectly coded to the helmet.
Fixes/removes/tweaks a lot of misinformation in the code about psion armors. 
Fixes psion heavy and light armor assuming genders. How rude.

Midly buffs the psion heavy armor slowdown from 2.25 down to 2. Very minor and might even deserve a touch more. My argument for this is the lack of armor modding like its sister the juggernaut.
Midly Buffs the psion heavy armor's helmet light to be slightly brighter to represent its very large new light.

Adds a icon for the psion heavy armor helmet light credit to myself yay!
Shifts some of these icon names around to make more sense.
Tested everything seems good. Look at dis cool new light tho!

dreamseeker_mM4BJb93Ek

### CODE REVIEW

File: code/modules/psionics/psionic_items/psi_Harmor.dm [hunk 1]
```diff
@@ -56,9 +56,8 @@
 	icon = 'icons/obj/psionic/occHicon.dmi'
 	icon_override = 'icons/obj/psionic/occHmob.dmi'
 	desc = "This is a massive helmet with a large and durable visor, looking through which you see the world through the veil of nothingness itself. You are detached from reality and it cannot harm you. You control reality, it doesn't control you. \
-	This is a very good defense, but maintaining it requires constant concentration."
+	This is a very good defense, but maintaining it requires constant concentration. It's massive visor serves as a more powerful Witchlight."
 	slot_flags = SLOT_HEAD
-	item_flags = STOPPRESSUREDAMAGE|THICKMATERIAL|AIRTIGHT|COVER_PREVENT_MANIPULATION|BLOCK_GAS_SMOKE_EFFECT
```
Comment (Trilbyspaceclone) [inline code/modules/psionics/psionic_items/psi_Harmor.dm:61]: intented?

File: code/modules/psionics/psionic_items/psi_Harmor.dm [hunk 2]
```diff
@@ -105,7 +104,6 @@
 	icon_state = "gloves"
 	icon_override = 'icons/obj/psionic/occHmob.dmi'
 	slot_flags = SLOT_GLOVES
-	item_flags = THICKMATERIAL
```
Comment (Trilbyspaceclone) [inline code/modules/psionics/psionic_items/psi_Harmor.dm:108]: Are you sure?

Comment (Yatiyaya) [reply code/modules/psionics/psionic_items/psi_Harmor.dm:108]: all psi armor has THICKMATERIAL, including the light/void armor, so heavy should too.

Comment (benj8560) [reply code/modules/psionics/psionic_items/psi_Harmor.dm:108]: intended? Yes and no. This one is correct...its slot flags where defined 2 times, Harmless but still bad clutter, However I removed both by mistake on some

Comment (benj8560) [reply code/modules/psionics/psionic_items/psi_Harmor.dm:108]: item flags*

Comment (benj8560) [reply code/modules/psionics/psionic_items/psi_Harmor.dm:108]: ah this is the gloves. Gloves simply dont need thickmaterial defined. It serves no function as far as I'm aware thickmaterial only does anything for chest/helmets

**Comment by benj8560** [comment]
aright should be all resolved. I was very eppy last night. Sleepy code

**Pull request merged**